batchflow/models/tf/unet.py

```diff
@@ -59,14 +59,35 @@ class UNet(EncoderDecoder):
     def default_config(cls):
         config = super().default_config()
 
-        config['initial_block'] = dict(layout='cna cna', kernel_size=3, filters=64)
+        config['initial_block'] = dict(layout='cna cna', kernel_size=3)
 
         config['body/encoder/num_stages'] = 4
-        config['body/encoder/blocks'] = dict(layout='cna cna', kernel_size=3, filters=[128, 256, 512, 1024])
+        config['body/encoder/order'] = ['downsampling', 'block']
+        config['body/encoder/blocks'].update(dict(layout='cna cna', kernel_size=3))
         config['body/embedding'] = None
-        config['body/decoder/blocks'] = dict(layout='cna cna', kernel_size=3, filters=[512, 256, 128, 64])
-
-        config['head'] = dict(layout='cna cna', kernel_size=3, filters=64)
+        config['body/decoder/order'] = ['upsampling', 'concat', 'block']
+        config['body/decoder/blocks'].update(dict(layout='cna cna', kernel_size=3))
 
         config['loss'] = 'ce'
         return config
+
+    def build_config(self, names=None):
+        config = super().build_config(names)
+
+        ibf = config.get('initial_block/filters')
```

Comment:
Do we need `initial_block` at all?
All the changes to `EncoderDecoder` were supposed to make it so we don't need it, and use `embedding` for the widest part (1024 filters) of the network.
This way there will be no shift in number of filters in `encoder`/`decoder`